build-shims.js

```diff
@@ -1,64 +1,64 @@
-const fs = require('fs');
-const path = require('path');
-const mkdirp = require('mkdirp');
-const mockery = require('mockery');
-const shimmedModules = require('./src/shims/index.js').SHIMMED_MODULES;
+const fs = require("fs");
```

Comment:
please revert change for ' to ".

Reply:
looks like some issue with editor's setup with prettier

Reply:
done